src/main/java/io/vertx/core/http/HttpVersion.java

```diff
@@ -25,5 +25,5 @@
  */
 @VertxGen
 public enum HttpVersion {
-  HTTP_1_0, HTTP_1_1
+  HTTP_1_0, HTTP_1_1, HTTP_2
```

Comment:
it is HTTP/2 : http://httpwg.org/specs/rfc7540.html

> On Apr 6, 2016, at 10:35 AM, Clement Escoffier <notifications@github.com> wrote:
> 
> In src/main/java/io/vertx/core/http/HttpVersion.java <https://github.com/eclipse/vert.x/pull/1353#discussion_r58669508>:
> 
> > @@ -25,5 +25,5 @@
> >   */
> >  @VertxGen
> >  public enum HttpVersion {
> > -  HTTP_1_0, HTTP_1_1
> > +  HTTP_1_0, HTTP_1_1, HTTP_2
> Should it be 2_0 ?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/eclipse/vert.x/pull/1353/files/bf875485313cf1f6ae199e2ab525539ed5676a2e#r58669508>